src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -124,7 +127,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
   const goToChatScreen = (channel: ChannelList, from) => {
     setChat({
       ...chat,
-      selectedChannel: channel
+      selectedChannel: channel as unknown as null
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `selectedChannel` property of the `chat` state is being assigned `null`. This could potentially lead to issues if other parts of your code expect this property to always be a `ChannelList` object. Make sure to add null checks wherever `selectedChannel` is used.
```diff
- selectedChannel: channel as unknown as null
+ selectedChannel: null
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->